SU-HSDO/suhumsci
# Shs 49 raised cards collection

## Summary
This adds a checkbox to the collection paragraph so that the 'raised cards' effect will automatically apply.

## Steps to Test
1. Pull down the branch, import config and clear the cache.
2. Create a flexible page. Add a collection. Verify there is the 'Raised Cards' option (screenshot below).
3. Check the raised cards box and in the collection add a View of Default Events (cards), a vertical timeline and a testimonial. 
4. Add another collection with a couple options from above but **do not** check the raised cards box.
5. View the page and verify the raised cards collection has the effect and the one without the checked box does not.
6. View the preexisting mixed raised cards page and make sure the work did not adversely affect this page (`/mixed-page-wells-raised-cards`)

## Screenshots
![Screen Shot 2022-11-08 at 9 03 11 AM](https://user-images.githubusercontent.com/8405274/200587726-e75af2fc-1d7b-414b-a6a2-6d329f35903e.png)

![Screen Shot 2022-11-08 at 9 12 51 AM](https://user-images.githubusercontent.com/8405274/200587742-e2807de6-51cc-4413-b6f9-56d4fcc5a444.png)

## Notes
The ticket requests some help on providing the help text for this option. I believe what I provided is clear, but feel free to add input.
